web_widget_text_markdown/static/tests/js/web_widget_text_markdown.js

```diff
@@ -0,0 +1,72 @@
+/* global QUnit */
+/* Copyright 2019 Alexandre Díaz - <dev@redneboa.es>
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_widget_text_markdown.test', function (require) {
+    'use strict';
+
+    var FormView = require('web.FormView');
+    var testUtils = require('web.test_utils');
+
+    var createAsyncView = testUtils.createAsyncView;
+
+
+    QUnit.module('fields', {
```

Comment:
Shouldn't you declare a new namespace for your tests?

```suggestion
    QUnit.module('web_widget_text_markdown', {
```